### PR TITLE
Create letters temp

### PR DESCRIPTION
**Pull request by Amrillo**
created letters template inside the letters page  and added required functionalities

### CODE REVIEW

File: src/app/components/letters/create-letters-temp/create-letters-temp.component.html [hunk 1]
```diff
@@ -1,10 +1,72 @@
 <app-top-bar></app-top-bar>
 <div class='container-crtLetter'>
-    <h1>Creating new letter's template</h1>
-    <div class='crtLetter__status'>
-      <strong>Name * </strong>
-      <mat-form-field appearance="outline">
-        <input matInput >
-      </mat-form-field>
-    </div>
+   <form class='crtLetter__form' [formGroup]='profileForm' (ngSubmit)='onSubmit()'>
```
Comment (AudriusBaltusis) [inline src/app/components/letters/create-letters-temp/create-letters-temp.component.html:3]: Please use double quotes in html, and single quotes in typescript files

File: src/app/components/letters/create-letters-temp/create-letters-temp.component.ts [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
 })
-export class CreateLettersTempComponent  {
+export class CreateLettersTempComponent {
+
+
+  constructor(private fb: FormBuilder,
+              private router: Router) {}
+
+   profileForm = this.fb.group( {
+     name: ["",[
```
Comment (AudriusBaltusis) [inline src/app/components/letters/create-letters-temp/create-letters-temp.component.ts:20]: Please use double quotes in html, and single quotes in typescript files

**Review by AudriusBaltusis — CHANGES_REQUESTED** [review]

File: src/app/components/letters/create-letters-temp/create-letters-temp.component.html (hunk 1 above)
Comment (Amrillo) [reply src/app/components/letters/create-letters-temp/create-letters-temp.component.html]: done

File: src/app/components/letters/create-letters-temp/create-letters-temp.component.ts (hunk 2 above)
Comment (Amrillo) [reply src/app/components/letters/create-letters-temp/create-letters-temp.component.ts]: done

**Review by AudriusBaltusis — APPROVED** [review]

**Pull request merged**